portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/Usage.testcase

```diff
@@ -48,12 +48,23 @@ definition {
 	}
 
 	@description = "Feature ID: Legacy | Automation ID: LRAC-8157 | Test Summary: Current add-ons are displayed in a card"
-	@ignore = "true"
 	@priority = "5"
 	test CurrentAddonsDisplayedInCard {
+		var propertyName = ACDXPSettings.connectDXPtoAnalyticsCloud();
+
+		ACUtils.launchAC();
+
+		ACProperties.switchProperty(propertyName = "${propertyName}");
+
+		ACNavigation.goToSettings();
 
-		// TODO LRAC-8157 CurrentAddonsDisplayedInCard pending implementation
+		ACSettings.goToUsage();
 
+		ACSettings.assertAddonsDetails(
```

Comment:
Hi @AfonsoHSantos , I would suggest you pass the values using a comma instead of periods, since they are in English US, they use this currency format, so you don't need to replace it on the macro.

Thanks

Reply:
Ooh, I see... you probably had problems when trying to escape the comma, since it is the delimiter for lists in Poshi...

Reply:
To solve this problem, please try to use ``\,`` to escape the commas that are considered the delimiter for lists in Poshi.

``
limitsIndividuals = "5\,000", 
limitsPageViews = "5\,000\,000",
``